core/src/main/java/org/apache/iceberg/rest/responses/ErrorResponse.java

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
This file is from https://github.com/apache/iceberg/pull/4201, and was only copied over to get the `ErrorHandlers` to compile.

It can be reviewed as part of https://github.com/apache/iceberg/pull/4201.